File: Java/SelectionSort/BJAlgo_2750.java
import java.io.*;

public class BJAlgo_2750 {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());
        int[] dataList = new int[N];
        for (int i = 0; i < N; i++) {
            dataList[i] = Integer.parseInt(br.readLine());
        }

        for (int stand = 0; stand < dataList.length - 1; stand++) {
            int lowest = stand;
            for (int num = stand + 1; num < dataList.length; num++) {
                if (dataList[lowest] > dataList[num]) {
                    lowest = num;
                }
            }
            int tmp = dataList[stand];
            dataList[stand] = dataList[lowest];
            dataList[lowest] = tmp;
        }

        for (int i = 0; i < dataList.length; i++) {
            System.out.println(dataList[i]);
        }
    }
}

File: Python/SelectionSort/BJAlgo_2750.py
from sys import stdin

N = int(stdin.readline())
data_list = list()
for _ in range(N):
    data_list.append(int(stdin.readline()))

for stand in range(len(data_list) - 1):
    lowest = stand
    for num in range(stand + 1, len(data_list)):
        if data_list[lowest] > data_list[num]:
            lowest = num
    data_list[stand], data_list[lowest] = data_list[lowest], data_list[stand]

for i in range(len(data_list)):
    print(data_list[i])